src/abstract/CSBondCore.sol

```diff
@@ -153,7 +153,9 @@ abstract contract CSBondCore is ICSBondCore {
         uint256 sharesBefore = LIDO.sharesOf(address(this));
         WSTETH.unwrap(amount);
         uint256 sharesAfter = LIDO.sharesOf(address(this));
-        _increaseBond(nodeOperatorId, sharesAfter - sharesBefore);
+        unchecked {
```

Comment:
it has `unchecked` internally (and `_unsafeReduceBond` as well)

Reply:
`unchecked` is being applied syntactically. So it has nothing to do with the internal `unchecked`s.